atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -436,6 +436,10 @@ def __init__(self, data, pulp_data, odcs_config, koji_tag=None, arches=None):
             if data.get(IGNORE_ABSENT_REPOS):
                 self.flags.append(IGNORE_ABSENT_REPOS)
 
+            dev_content_sets = data.get('dev_content_sets', {})
+            for arch, cont_sets in dev_content_sets.items():
+                self.pulp[arch] = set(cont_sets).union(self.pulp.get(arch, []))
```

Comment:
```suggestion
                self.pulp.setdefault(arch, set()).union(cont_sets)
```

Reply:
self.pulp is list not set, so this won't work